README.md

```diff
@@ -53,9 +53,31 @@ Combining `editor.suggest.preview` with the [pre-fill required fields](#code-com
 
 Completing the snippet allows you to tab complete through each attribute and block.
 
-### Syntax Validation
+### Validation
```

Comment:
FYI changing headers like this breaks the generated URL link, so if we (or anyone else) linked to this markdown block it won't work. Not the end of the world, as it'll load the page and just stop at the top, but thought I should mention it.

Reply:
Good point - I think I put too much emphasis on the word "syntax" here when it _can_ include pretty much all that we do (not just HCL syntax).

I will change it back.

I am hoping that at some point in the future this long list is split into individual files per feature and we publish the whole thing as a website - at which point I can imagine link stability becoming more important.